traffic_portal/app/src/common/modules/form/deliveryService/form.deliveryService.DNS.tpl.html

```diff
@@ -746,7 +746,7 @@ <h3>Current Value</h3>
                         </label>
                         <div class="col-md-10 col-sm-10 col-xs-12">
                             <small class="input-warning" ng-show="!settings.isNew && routingConfig.routingName.$dirty">Warning: Changing the routing name may require SSL certificates to be updated.</small>
```

Comment:
you can get rid of these warnings on all the routing name fields that should no longer be relevant

Reply:
Aha.. that makes sense.., i'll do that.